slrealizer/realize_om10.py

```diff
@@ -0,0 +1,342 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from realize_sl import SLRealizer
+from utils.constants import *
+from utils.utils import *
+import numpy as np
+import pandas as pd
+import galsim
+import gc # need this to optimize memory usage
+
+class OM10Realizer(SLRealizer):
+
+    """
+    
+    A class that realizes objects in the OM10 mock quasar catalog
+    under the given observation conditions, 
+    into LSST DRP Source and Object catalogs
+    
+    """
+    
+    def __init__(self, observation, catalog, debug=False, add_moment_noise=True, add_flux_noise=True):
+        #super(OM10Realizer, self).__init__(observation) # Didn't work for some reason
+        self.as_super = super(OM10Realizer, self)
+        self.as_super.__init__(observation, add_moment_noise=add_moment_noise, add_flux_noise=add_flux_noise)
+        self.catalog = catalog
+        self.num_systems = len(self.catalog.sample)
+        self.DEBUG = debug
+        
+    def get_lens_info(self, objID=None, rownum=None):
+        if objID is not None and rownum is not None:
+            raise ValueError("Need to define either objID or rownum, not both.")
+
+        if objID is not None:
+            return self.catalog['objectId'] # Don't use this (?)
+        elif rownum is not None:
+            return self.catalog.sample[rownum]
+
+    def _om10_to_galsim(self, lens_info, band):
+        """
+        Converts OM10's column values into GalSim terms
+        
+        Keyword arguments:
```

Comment:
I recommend `numpydoc` docstring style, as here: https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
We can switch later, better to merge this PR soon!

Reply:
Ohh, I assumed it was the same as PEP 257: https://www.python.org/dev/peps/pep-0257/

Reply:
numpydoc goes beyond this: you'll see why it's good once we have the API docs up on RTD.